modules/touch/touchmodule.cpp

```diff
@@ -114,9 +120,8 @@ bool TouchModule::hasNewInput() {
 }
 
 void TouchModule::hasNewWebInput(const std::vector<TuioCursor>& listOfContactPoints) {
```

Comment:
Suggest renaming this to `processNewWebInput` to be consistent with your new naming of `processNewInput` above.

Reply:
Good point, fixed now.